statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1309,7 +1309,7 @@ def resid_pearson(self):
     @cache_readonly
     def resid_working(self):
         # Isn't self.resid_response is already adjusted by _n_trials?
-        val = (self.resid_response / self.family.link.deriv(self.mu))
+        val = (self.resid_response * self.family.link.deriv(self.mu))
         val *= self._n_trials
```

Comment:
Wait, I was thinking we didn't need `_n_trials`. I think `resid_response` is already adusted.